src/Microsoft.Health.Fhir.Liquid.Converter/DotLiquids/MemoryFileSystem.cs

```diff
@@ -5,16 +5,23 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using DotLiquid;
 using DotLiquid.FileSystems;
 using Microsoft.Health.Fhir.Liquid.Converter.Exceptions;
 using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Microsoft.Health.Fhir.Liquid.Converter.Utilities;
 
 namespace Microsoft.Health.Fhir.Liquid.Converter.DotLiquids
 {
     public class MemoryFileSystem : ITemplateFileSystem
```

Comment:
>MemoryFileSystem [](start = 17, length = 16)

suggest to add a new class 'TemplateLocalFileSystem'

Reply:
Thanks, fixed in the update.